### DISCUSSION TITLE
Is there a size limit in solving the robust problem? 

### DISCUSSION THREAD

**Discussion by NewWalker2020** [yalmip/YALMIP#1158] [General discussion] [answered]
When I try to solve a MIP robust problem setting uncertain variable with the "uncertain" command in yalmip, it seems having some trouble in model transformation. I guess that yalmip may have a size limit in solving the robust problem diretly. If so, can u tell me the maximum scale of the robust problem that yalmip supports? or how can I avoid this problem?
It has been stuck in this step for a day! I am a greenhand in yalmip and don't konw what to do next.....
![robust problem](https://user-images.githubusercontent.com/64523019/182386241-33d4a21e-b4d9-4ab1-b1ed-d2c2ed481fd9.png)
![robust](https://user-images.githubusercontent.com/64523019/182386275-179b9c2c-9c1e-43a2-a841-52c8451eb12d.png)

**Comment by johanlofberg** [comment, marked as answer]
YALMIP does not have any limit, but many robust optimization problems are completely intractable as they grow exponentially in size

**Reply by NewWalker2020** [reply]
ok, I get it. Thanks for your reply!
